lib/exception_handling.rb

```diff
@@ -595,6 +596,8 @@ def send_exception_summary( exception_data, first_seen, occurrences )
         deliver(ExceptionHandling::Mailer.exception_notification(exception_data, first_seen, occurrences))
       end
     rescue StandardError, MailerTimeout => ex
+      exception_context = exception_data[:environment]
+      exception_or_string = exception_data[:error_string]
       $stderr.puts("ExceptionHandling.log_error_email rescued exception while logging #{exception_context}: #{exception_or_string}:\n#{ex.class}: #{ex}\n#{ex.backtrace.join("\n")}")
```

Comment:
While we are here, can you factor out the portion of this error string that is common?  
```
"ExceptionHandling.log_error_email rescued exception while logging #{exception_context}: #{exception_or_string}
```